docs/dev/news/explicit-failing-assertions.feat

```diff
@@ -0,0 +1,5 @@
+Code actions in editor to explicit failing assertions.
+In VSCode, place the cursor on a failing division by zero and either
```

Comment:
Since you've added more cases, you should generalize this a bit! Perhaps say that several common assertion types are supported with more on the way, and mention what happens if one isn't supported (?)